_data/toc.yaml

```diff
@@ -1275,10 +1277,22 @@ reference:
     section:
     - path: /edge/engine/reference/commandline/trust/
       title: docker trust
+    - path: /edge/engine/reference/commandline/trust_key/
+      title: docker trust key
+    - path: /edge/engine/reference/commandline/trust_key_generate/
```

Comment:
These need to be "under" `docker trust key`. Same issues with `trust signer` sub-subcommands. @riyazdf this is not a pattern I've seen before. Is it OK for us to have this extra level of sub-command depth? cc/ @thaJeztah

Reply:
@mistyhacks: I'm fine with another directory level if it's preferred